hook.cpp

```diff
@@ -332,7 +332,7 @@ Up        r    sub_4649C0+29  call    ds:SetCursorPos
 	PATCH_PUSH_STRING_PTR_VALUE(0x4064C1, "1.4");
 	PATCH_PUSH_STRING_PTR_VALUE(0x4346DE, __DATE__ __TIME__);
```

Comment:
Can you fix this one too in another commit to this PR?

Reply:
Sure